.github/workflows/beta-publish.yml

```diff
@@ -14,28 +14,19 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v2
 
-      - name: Cache NPM files
-        uses: actions/cache@v2
-        with:
-          path: ~/.npm
-          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
-          restore-keys: |
-            ${{ runner.os }}-node-
-
-      - name: Cache Node Modules
-        id: cache
-        uses: actions/cache@v2
-        with:
-          path: node_modules
-          key: node-modules-${{ hashFiles('package-lock.json') }}
-
       - name: Setup Node
         uses: actions/setup-node@v1
         with:
           node-version: "12"
           registry-url: "https://npm.pkg.github.com"
 
-      - name: Clean install dependencies if lockfile changed
+      - name: Cache Node files
```

Comment:
needs an `id: cache`

Reply:
otherwise the npm ci will always run

Reply:
https://github.com/WPMedia/themes-lerna-playground/pull/15/files#diff-3928becfcdf7981d7c15d4a8714a11384ecbdbda50fa5abb08cd04e456678414R24

Reply:
Thanks! got it!